demos/device_defender_for_aws/metrics_collector/lwip/metrics_collector.c

```diff
@@ -0,0 +1,254 @@
+/* Standard includes. */
+#include <stdint.h>
+/* Lwip includes. */
+#include "lwip/arch.h"
+#include "lwip/stats.h"
+#include "lwip/tcpip.h"         /* #define LOCK_TCPIP_CORE()     sys_mutex_lock(&lock_tcpip_core) */
+#include "lwip/ip_addr.h"       /* ip_addr_t, ipaddr_ntoa, ip_addr_copy */
+#include "lwip/tcp.h"           /* struct tcp_pcb */
+#include "lwip/udp.h"           /* struct udp_pcb */
+#include "lwip/priv/tcp_priv.h" /* tcp_listen_pcbs_t */
+/* Platform layer includes. */
+#include "platform/iot_threads.h"
+/* Demo logging include. */
+#include "iot_demo_logging.h"
```

Comment:
We are trying to get away from logging file.

Reply:
Done by dbfa0fc8c562ae295a49aa8905ac5c92457a91b0